src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -554,6 +555,18 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
 
                             continue;
 
+                        case "sourcelink":
+                            value = RemoveQuotesAndSlashes(value);
```

Comment:
Does the sourcelink parameter accept files in sub-folder (or parent folders)?
Do we need to worry about unix vs windows path formats here?